test/acceptance/member_test.exs

```diff
@@ -18,19 +18,4 @@ defmodule Community.Acceptance.MemberTest do
     assert last_member.company_name == "X-men"
     assert last_member.approved == false
   end
-
-  defp click_on(text) do
-    find_element(:link_text, text)
-    |> click
-  end
-
-  defp click_role(name) do
```

Comment:
I may pull out the extract module.

It's probably a bit premature to extract that but i rely on clicking data roles heavily enough that I'm sure it will come up again soon.